src/browserless.ts

```diff
@@ -342,8 +359,6 @@ export class BrowserlessServer {
     await this.puppeteerProvider.close();
 
     debug(`Successfully shutdown, exiting`);
-
-    process.exit(0);
```

Comment:
We still might need this to force the process to close (unless lightship does this somehow?)

Reply:
It does, AFAIK.  
> Do not call process.exit() in a shutdown handler – Lighthouse calls process.exit() after all registered shutdown handlers have run to completion.